src/app/models/timeslot.rb

```diff
@@ -25,7 +25,7 @@ def initialize(start, stop)
     end
 
     def to_s(with_day: false)
-      "#{start_day + ' ' if with_day}#{start_time} - #{end_time}"
+      "#{start_day + ' ' if with_day}#{start_time} – #{end_time}"
```

Comment:
Yeah, but I don’t think that’s much clearer than just being able to see the actual char. Sends people to look up the hex code. I’m wishing for a human-readable `\N{{en dash}}` or something like that.